kibana.md

```diff
@@ -1042,6 +1042,12 @@ Open `config/kibana.yml` and append the following:
 readonlyrest_kbn.login_custom_logo: 'https://.../logo.png'
 ```
 
+To load your custom logo directly from the file, place the logo into the `<YOUR_ROOT_DIRECTORY>/kibana/plugins/readonlyrestkbn/public/assets` folder and append the following
```

Comment:
asked chatgpt to make the sentence flow beetter:

```
To incorporate your personalized logo into the login page, place your image file within the <YOUR_ROOT_DIRECTORY>/kibana/plugins/readonlyrestkbn/public/assets directory. Then, proceed by appending the following code snippet to `kibana.yml`:
```

Can you spend a few words about the admitted logo file formats and sizes?

Reply:
> asked chatgpt to make the sentence flow better:

Rephrased it ✔️ 

> Can you spend a few words about the admitted logo file formats and sizes?

I added the info, but generally, it depends on the browser